class.jetpack.php

```diff
@@ -2834,6 +2837,22 @@ public static function get_additional_stat_data( $prefix = '' ) {
 		return $return;
 	}
 
+	private static function get_site_user_count() {
+		global $wpdb;
+
+		if ( function_exists( 'wp_is_large_network' ) ) {
+			if ( wp_is_large_network( 'users' ) ) {
+				return 10101; // Not a real value but should tell us that we are dealing with a large network.
```

Comment:
Oh yeah, I don't know why I got the impression that it was just a query wrapped in a transient - the number is generated elsewhere.

When IS the number generated?